CONTRIBUTING.md

```diff
@@ -15,6 +15,10 @@ Node.js software.
 For general help using Node.js, please file an issue at the
 [Node.js help repository](https://github.com/nodejs/help/issues).
 
+For help working with the vendored version of V8 included in Node's dependency
```

Comment:
for instructions on updating the version of V8 included in node's dependency directory...